pkg/ip/addr.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
Out of curiosity, does Windows also need to worry about "tentative" addresses?

Reply:
Not sure, asking around

Reply:
So it seems Windows doesn't need to worry about it right now since HNS doesn't support IPv6 yet - so ja bitte for now I guess ;-)